src/style.css

```
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+form {
+  margin-top: 20px;
+}
+
+input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

set the font-family to `html` selector and use `inherit` value here
```suggestion
  font-family: inherit;
```